applications/vessel-history/src/features/vessel-list-item/VesselListItem.tsx

```diff
@@ -19,31 +40,43 @@ const VesselListItem: React.FC<ListItemProps> = (props): React.ReactElement => {
           className={styles.deleteSaved}
         ></IconButton>
       )}
-      <h3>Don Ying Gang 8003</h3>
+      <h3>{vessel?.name}</h3>
       <div className={styles.identifiers}>
         <div>
           <label>FLAG</label>
-          Chinese Taipei
-        </div>
-        <div>
-          <label>MMSI</label>
-          31306526
-        </div>
-        <div>
-          <label>IMO</label>
-          12345
+          {flagLabel}
         </div>
+        {vessel.mmsi && vessel.mmsi.length && (
+          <div>
+            <label>MMSI</label>
+            {vessel.mmsi[0].value}
+          </div>
+        )}
+        {vessel.imo && vessel.imo !== '0' && (
+          <div>
+            <label>IMO</label>
+            {vessel.imo}
+          </div>
+        )}
+        {vessel.callsign && vessel.callsign.length && (
+          <div>
+            <label>CALLSIGN</label>
+            {vessel.callsign[0].value}
+          </div>
+        )}
         <div>
-          <label>CALLSIGN</label>
-          407201
+          <label>TYPE</label>
+          {vessel.type || 'Unknown'}
         </div>
+        {getGearType(vessel) && (
+          <div>
+            <label>GEAR TYPE</label>
+            {getGearType(vessel)}
+          </div>
+        )}
         <div>
           <label>TRANSMISSIONS</label>
-          from 2015-07-21 to 2020/04/13
-        </div>
-        <div>
-          <label>IMO</label>
-          12345
+          from {vessel.firstTransmissionDate} to {vessel.lastTransmissionDate}
```

Comment:
Should dates be formatted?

Reply:
no, is the same format of the design, and not have timezone offset